### PR TITLE
raghav's card

### PR DESCRIPTION
**Pull request by viper-raghav**
hey
this is my first opensource project and i'm very eager to do more in future
i hope you like it.
thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,7 +114,32 @@ <h2>A project for first-time contributions</h2>
...
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-		
+
+		<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->  
+    <div class="card">
+      <p class="name">Raghav</p>
+      <p class="contact">
+        <i class="fab fa-twitter"></i>
+        <a href="https://twitter.com/raghavu37260366" target="_blank">raghavu37260366</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
        <a href="https://twitter.com/raghavu37260366" target="_blank">@raghavu37260366</a>
```

File: index.html [hunk 2]
```diff
@@ -114,7 +114,32 @@ <h2>A project for first-time contributions</h2>
...
+      </p>
+      <p class="about">hey there, i'm a first year b.tech student learing basic programming. i like to read to read books mostly biographies.</p>
+      <div class="resources">
+        <p>3 Useful Dev Resources</p>
+        <ul>
+          <li>
+            <a href="https://youtube.com/playlist?list=PLu0W_9lII9agiCUZYRsvtGTXdxkzPyItg" target="_blank" title="it is a free web devlopment course by codewithharry">Resource 1</a>
+          </li>
+          <li>
+            <a href="#" target="_blank" title="Second Resource">https://youtube.com/playlist?list=PLfqMhTWNBTe3H6c9OGXb5_6wcc1Mca52n</a>
```
Comment (TenTraicion) [inline index.html:133]:
```suggestion
            <a href="https://youtube.com/playlist?list=PLfqMhTWNBTe3H6c9OGXb5_6wcc1Mca52n" target="_blank" title="Second Resource">web development course by Apna College</a>
```

File: index.html [hunk 3]
```diff
@@ -114,7 +114,32 @@ <h2>A project for first-time contributions</h2>
...
+          </li>
+          <li>
+            <a href="#" target="_blank" title="Third resource">https://youtube.com/playlist?list=PL-Jc9J83PIiEeD3I4VXETPDmzJ3Z1rWb4</a>
```
Comment (TenTraicion) [inline index.html:136]:
```suggestion
            <a href="https://youtube.com/playlist?list=PL-Jc9J83PIiEeD3I4VXETPDmzJ3Z1rWb4" target="_blank" title="Third resource">web development course by pepcoding</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@viper-raghav, Thanks for your contribution. Kindly do the recommended changes and commit. Happy Coding.

**Comment by viper-raghav** [comment]
thanks

**Comment by viper-raghav** [comment]
thanks

**Comment by viper-raghav** [comment]
thanks

**Comment by viper-raghav** [comment]
thanks

**Comment by viper-raghav** [comment]
thanks

**Comment by viper-raghav** [comment]
thanks

**Pull request closed without merging**